tools/lint.js

```diff
@@ -19,7 +19,8 @@ if (Deno.args.includes("--rs")) {
 if (!didLint) {
   await Promise.all([
     dlint(),
-    dlintPreferPrimordials(),
+    // todo(dsherret): re-enable
+    // dlintPreferPrimordials(),
```

Comment:
FYI @petamoriken -- I temporarily disabled this because there were about 50 lint errors.

Reply:
OK, I'll check it.